### ISSUE TITLE
Does not work with Confluence Server

### ISSUE THREAD

**Issue by BrianKrohn** [anth0d/chrome-confluence-search#4]
I set the URL to my company's Confluence site, but the extension errors and says I need to log into my site, even though I already am.

**Comment by anth0d** [comment]
Thanks for the issue report and I apologize for not responding sooner.

Can you right-click inside the extension and select "Inspect" and let me know what error (if any) is reported in the DevTools Console? Copy and paste it out before closing the extension. 

If that doesn't show anything, there is another tab in DevTools (called Network) which should show the Network request/response. That will help me identify what is going wrong.

**Comment by andrewgl504** [comment]
I'm currently having the same issue. 

I enter the URL, and type in my URL, and receive the error "Error! Visit your site and log in." I'm logged in, so I can't fix that.

If I right click the extension and click Inspect Popup, I do not see any obvious errors under DevTools Console or DevTools Network.

If I wait for the popup to appear, and right click that, the Inspect menu option is grayed out as unselectable.

**Comment by anth0d** [comment]
Thanks @andrewgl504, can you look in DevTools and check the "Preserve log" checkbox in the Network tab, and in the Console tab (under the gearwheel icon)? 

Then try to reproduce the error and let me know if you see anything logged.

**Comment by andrewgl504** [comment]
No luck. 

Here's a screen grab of how I'm using it: https://recordit.co/N1vO1PppX5

**Comment by anth0d** [comment]
I fixed another issue that was causing the extension to incorrectly show a search error, once the `1.1.3` version is approved and published by Chrome Web Store (currently pending) this issue should be resolved. 

@andrewgl504 one thing I see in that screencast is that it looks like you need to add `/wiki` to the end of your URL. 

Again... I seriously apologize for missing this thread entirely. I've now updated my settings to send me emails when comments like these come in, so let me know if you have any issues trying this out again and I will be quick to respond. I do hope you'll find it useful, I personally use it every day for my work. 

https://chrome.google.com/webstore/detail/confluence-quick-search/gimcmmlpmjffkpbomagapjhdfbbeldfk

**Comment by andrewgl504** [comment]
I've updated the extension, and set the URL (see the below screenshot). Still getting an error.

![Screen Shot 2021-03-22 at 11 20 19 AM](https://user-images.githubusercontent.com/12555259/112022764-97ebb580-8b00-11eb-868e-e4aaf686c957.png)

**Comment by anth0d** [comment]
Can you quickly try without the `/` at the end, i.e. `/wiki`.

Also, open the popup, right-click the background and click "Inspect". This
should open a separate Chrome DevTools window outside of your browser
window. From here when your search fails there should be an error logged to
the console. If you do not see anything, its possible you do not have the
popup's DevTools open.

**Comment by andrewgl504** [comment]
Removing the trailing slash worked! Thought I had tried that, thanks!

Looking forward to using this.

**Comment by anth0d** [comment]
Awesome news. Again, sorry for the delayed response. Hope you find it useful. I recommend setting a keybinding, [chrome://extensions/shortcuts](chrome://extensions/shortcuts) -- I use `Command-Shift-L` personally.

I will fix the issue that a trailing slash should be detected/removed automatically.
